src/docs/markdown/caddyfile/common-uses.md

```diff
@@ -111,11 +110,18 @@ rewrite /add     /add/
 rewrite /remove/ /remove
```

Comment:
```suggestion
rewrite /add     /add/
rewrite /remove/ /remove
```

Reply:
Is that not https://github.com/caddyserver/website/pull/42/files#diff-9f61e9f747cde1dd3e210174ef80eda0R109 ?

Reply:
The heck, it didn't show up for me 🤷‍♂️